feature-about/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This should be moved to `core.model/commonMain/resources`

Reply:
In "MR" directly right?

Reply:
Yes!